chat_server/blueprints/personas.py

```diff
@@ -112,6 +113,7 @@ async def add_persona(
     if existing_model:
         raise DuplicatedItemException
     MongoDocumentsAPI.PERSONAS.add_item(data=request_model.model_dump())
+    await notify_personas_changed(request_model.supported_llms)
```

Comment:
Add a note that this implementation will not scale and a better solution might be to watch the database for changes instead of triggering on rest endpoints.

https://search.brave.com/search?q=mongodb+watch+document+for+changes+python&source=web&summary=1&conversation=ff1b4bcd8705abedf025a2